kolibri/plugins/hooks.py

```diff
@@ -234,6 +234,28 @@ class and all parent abstract classes - can only be called on an abstract
     return subclass
 
 
+def unregister_hook(subclass):
+    """
+    This method must be used as a decorator to unregister a hook and prevent it from being
+    registered against the abstract hook classes it inherits from.
+    """
+    if not any(
+        hasattr(base, "_registered_hooks")
+        and base.abstract
+        and issubclass(base, KolibriHook)
+        for base in subclass.__bases__
+    ):
```

Comment:
Same comment here regarding consolidation. Seems this validation logic is pretty much the same as `register_hook`